.circleci/config.yml

```diff
@@ -5,7 +5,7 @@
 # Adopted from
 # https://github.com/facebookresearch/detectron2/blob/master/.circleci/config.yml
 
-version: 2
+version: 2.1
```

Comment:
Does the workflows version also need to change?

Reply:
yeah, it seems to be needed due to parameterization in the config.